hpi-schul-cloud/nuxt-client
# BC-4455 - Injection error for "envConfigModule"

# Short Description
QA note: 
Change tested, no issues found, for more details see jira ticket: https://ticketsystem.dbildungscloud.de/browse/BC-4455


@christian-darsow pointed out to us, that by https://github.com/hpi-schul-cloud/nuxt-client/pull/2607/commits/74e1a1e0a01e2eb5eba8d4d79a172a446fac976a we introduced an injection error within the "LanguageMenu".



## Links to Ticket and related Pull-Requests


https://ticketsystem.dbildungscloud.de/browse/BC-4455

Link to deployment : https://default-bc-4455-injection-error-for-envconfigmodule.cd.dbildungscloud.dev/

@SevenWaysDP original changes : https://github.com/hpi-schul-cloud/nuxt-client/pull/2607/commits/74e1a1e0a01e2eb5eba8d4d79a172a446fac976a

## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
